### DISCUSSION TITLE
Display routing service

### DISCUSSION THREAD

**Discussion by airald** [devemux86/cruiser#184] [Q&A] [answered]
Just a suggestion. I think that should be interesting to display the current routing service in Routing menu.

**Comment by devemux86** [comment]
@airald indeed this would be useful info!

Something like this?

<img src="https://github.com/devemux86/cruiser/assets/3484020/037ea1f3-c03e-42cd-814d-2819d127a4fe" width="50%"/>

**Reply by airald** [reply]
Exactly this!

**Comment by devemux86** [comment, marked as answer]
Display of routing service is available in [Cruiser 3.0.21](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
